mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/ConfirmCompositeSwapScreen.tsx

```diff
@@ -100,6 +103,22 @@ export function ConfirmCompositeSwapScreen ({ route }: Props): JSX.Element {
     }
   }
 
+  function getTransactionType (): string {
+    if (conversion?.isConversionRequired === true) {
+      if (!isFutureSwap) {
+        return translate('screens/ConfirmCompositeSwapScreen', 'Convert & swap')
+      } else {
+        return translate('screens/ConfirmCompositeSwapScreen', 'Convert & future swap')
+      }
+    } else {
+      if (!isFutureSwap) {
+        return translate('screens/ConfirmCompositeSwapScreen', 'Swap')
+      } else {
+        return translate('screens/ConfirmCompositeSwapScreen', 'Future swap')
+      }
+    }
+  }
+
```

Comment:
Might not have big impact, but let's still apply fair early approach since most of the time people will use this for normal swap without conversion. 

Should we remove the **Convert and future swap** since there's no `DFI` in future swap?


```suggestion
  function getTransactionType(): string {
    if (conversion?.isConversionRequired !== true) {
      if (!isFutureSwap) {
        return translate('screens/ConfirmCompositeSwapScreen', 'Swap')
      } else {
        return translate('screens/ConfirmCompositeSwapScreen', 'Future swap')
      }
    } else {
      return translate('screens/ConfirmCompositeSwapScreen', 'Convert & swap')
    }
  }
  
```

```

Reply:
yep make sense, will update

Reply:
updated